Sources/MintKit/Mint.swift

```diff
@@ -406,6 +406,46 @@ public class Mint {
         output("Installed \(packageCount) from \(mintFilePath.string)".green)
     }
 
+    public func dump(force: Bool) throws {
+        guard mintFilePath.absolute().exists == false || force else {
+            throw MintError.couldNotOverwriteMintfile(mintFilePath.string)
+        }
+
+        guard packagesPath.exists else {
+            output("No mint packages installed")
+            return
+        }
+
+        var packagesToExport: [String] = []
+
+        let packages: [Path] = try packagesPath.children().filter { $0.isDirectory }
+
+        let metadata = try readMetadata()
+
+        // find packages
+        try packages.forEach { (packagePath) in
+            let packagePathName = packagePath.lastComponent
+            let package = metadata.packages.first { $0.value == packagePathName }!
+            let packageRepo = package.key
+
+            let versions = try (packagePath + "build")
+                .children()
+                .filter { !$0.lastComponent.hasPrefix(".") }
+                .sorted()
+                .map { $0.lastComponent }
+
+            for version in versions {
```

Comment:
This would write all the versions you have installed. Presumably you would just want the latest one right?

Reply:
Well my thought was that since with Mint you can have multiple installed versions and since it is a sub command of the list comand that displays all the versions I would be expecting to get all of the installed in the Mintfile as well. So I can also install them later.

But I have no strong opinion on this. Thus I leave the decision to you, would you like this changed?

Reply:
@yonaskol any input on this? :)